modules/compliance-anatomy.adoc

```diff
@@ -254,7 +255,7 @@ The example scan has reported some findings. One of the remediations can be enab
 
 [source,terminal]
 ----
-$ oc patch complianceremediations/rhcos4-e8-worker-audit-rules-dac-modification-chmod --patch '{"spec":{"apply":true}}' --type=merge
+$ oc patch complianceremediations/rhcos4-e8-audit-rules-dac-modification-chmod --patch '{"spec":{"apply":true}}' --type=merge
```

Comment:
No need to change.
`$ oc patch complianceremediations/rhcos4-e8-worker-audit-rules-dac-modification-chmod --patch '{"spec":{"apply":true}}' --type=merge` is correct.
For complianceremediations for node type rules, the naming rule should be <profile name>+<master/worker>+<rule name_without ocp4/rhco4 prefix>.
```
$ oc get cr --show-labels 
NAME                                                        STATE        LABELS
rhcos4-e8-worker-audit-rules-dac-modification-chmod         NotApplied   compliance.openshift.io/scan-name=rhcos4-e8-worker,compliance.openshift.io/suite=rhcos4-e8-worker
rhcos4-e8-worker-audit-rules-dac-modification-chown         NotApplied   compliance.openshift.io/scan-name=rhcos4-e8-worker,compliance.openshift.io/suite=rhcos4-e8-worker
...
$ oc get cr --show-labels 
NAME                                                        STATE        LABELS
rhcos4-e8-master-audit-rules-dac-modification-chmod         NotApplied   compliance.openshift.io/scan-name=rhcos4-e8-master,compliance.openshift.io/suite=test
rhcos4-e8-master-audit-rules-dac-modification-chown         NotApplied   compliance.openshift.io/scan-name=rhcos4-e8-master,compliance.openshift.io/suite=test
```